src/constants/constants.ts

```diff
@@ -55,6 +66,7 @@ export const gettingStartedGuideLink = 'https://aka.ms/mssql-getting-started';
 export const changelogLink = 'https://aka.ms/vscode-mssql-changelog';
 export const integratedAuthHelpLink = 'https://aka.ms/vscode-mssql-integratedauth';
 export const sqlToolsServiceCrashLink = 'https://github.com/Microsoft/vscode-mssql/wiki/SqlToolsService-Known-Issues';
+export const databaseString = 'Database';
 
```

Comment:
I'd rather see this as a string enum here at least so we know what it's intended use is and makes it easier to discover the nodeTypes that are supported.